File: wej-wyj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wej-wyj.py
#  


def main(args):
    imie = input('Podaj swoje imię\n')
    nazwisko = input('Podaj swoje nazwisko\n')
    print('Witaj ', imie, nazwisko, '!')

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: CPP/wej-wyj.cpp
/*
 * wej-wyj.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    char imie[20];
    char nazwisko[20];
    
    cout << "Podaj imię: ";
    cin >> imie;
    cout << "Podaj nazwisko: ";
    cin >> nazwisko;
    cout  << "Witaj " << imie << nazwisko << endl;
    
    return 0;
}

